product-mini/platforms/linux/CMakeLists.txt

```diff
@@ -148,7 +150,7 @@ add_executable (iwasm main.c ${UNCOMMON_SHARED_SOURCE})
 
 install (TARGETS iwasm DESTINATION bin)
 
-target_link_libraries (iwasm vmlib ${LLVM_AVAILABLE_LIBS} ${UV_A_LIBS} -lm -ldl -lpthread)
+target_link_libraries (iwasm vmlib ${LLVM_AVAILABLE_LIBS} ${UV_A_LIBS} tensorflow-lite -lm -ldl -lpthread)
 
 add_library (libiwasm SHARED ${WAMR_RUNTIME_LIB_SOURCE})
```

Comment:
We should create this CMakelist in our wasi-nn test folder. We don't want to modify it here because otherwise all the people that uses now WASI-NN will be downloading tensorflow

Reply:
pushed